app/menu.js

```diff
@@ -1,11 +1,15 @@
+'use strict';
```

Comment:
yes, it's OK now since hyper is running on node 7.4 since https://github.com/zeit/hyper/pull/1848 👌